packages/nuxt/src/server/sdk.ts

```diff
@@ -14,6 +14,17 @@ export function init(options: SentryNuxtServerOptions): Client | undefined {
   const sentryOptions = {
     ...options,
     registerEsmLoaderHooks: mergeRegisterEsmLoaderHooks(options),
+    integrations: [
+      httpIntegration({
+        instrumentation: {
+          responseHook: () => {
+            // Makes it possible to end the tracing span before closing the Vercel lambda (https://vercel.com/docs/functions/functions-api-reference#waituntil)
```

Comment:
q: How about netlify and others? I wonder if we also want to flush in those cases?

Reply:
As far as I know Netlify does not expose a function like `waitUtil` 🤔  But it might make a difference if the SDK just flushes in the response hook - I would have to try.

Reply:
Right, maybe we can try flushing here in case of serverless.